### ISSUE TITLE
LE Scan

### ISSUE THREAD

**Issue by colemancda** [PureSwift/BluetoothLinux#14]
Implement LE Scan.
First step is to implement the "LE Set Scan Enable Command" PDU.

Reference the [Bluetooth Core Specification v4.0](https://drive.google.com/file/d/0BxhBhhqJ0fN3dy05OXc0U2dtRDA/view?usp=sharing) page 823 (7.8.11 LE Set Scan Enable Command)

**Comment by semireg** [comment]
Has any progress been made on this? If not, I'd like to give it a shot. I've successfully built this project on an Ubuntu VM. Is it possible to build on OSX using the dongle connected to the Mac? Having a native Xcode environment would certainly speed up development. This is what I'm seeing today when I run `swift build`

```
BluetoothLinux-master/Sources/BluetoothLinux/POSIXError.swift:34:39: error: use of unresolved identifier 'NSPOSIXErrorDomain'
        let nsError = NSError(domain: NSPOSIXErrorDomain, code: Int(code.rawValue), userInfo: nil)
                                      ^~~~~~~~~~~~~~~~~~
Foundation.NSPOSIXErrorDomain:1:12: note: did you mean 'NSPOSIXErrorDomain'?
public let NSPOSIXErrorDomain: String
           ^
<unknown>:0: error: build had 1 command failures
```

**Comment by colemancda** [comment]
I use Parallels and compile on Ubuntu, and edit on Mac in Xcode. Only Bluetooth will compile on Mac, the SPM support is for Linux only.

**Comment by colemancda** [comment]
@semireg Implemented in [v2.5.0](https://github.com/PureSwift/BluetoothLinux/releases/tag/2.5.0)
